integration_tests/steps/power_tests.py

```diff
@@ -26,3 +26,20 @@ def test_export_of_power_flow_result(context):
     sim_data_csv = glob.glob(os.path.join(context.export_path, "*", "plot", "power_flow.html"))
     if len(sim_data_csv) != 1:
         raise FileExistsError("Not found in {path}".format(path=context.export_path))
+
+
+@then('BaselinePeakEnergyStats are correctly calculated')
+def test_baseline_peak_energy_stats(context):
+    area_throughput_stats = \
+        context.simulation.endpoint_buffer.area_throughput_stats.results
+
+    expected_results = {'Neighborhood 1': {'import': {'peak_energy_kWh': 0.4,
+                                                      'peak_percentage': 100.0}},
+                        'House 1': {'import': {'peak_energy_kWh': 0.4,
```

Comment:
To maintain the uniformity JSON structure, wouldn't it be better to always have the same JSON structure, for instance in this case `["House 1"]["export"]["peak_energy_kWh"] = 0` rather than silently having no key at all?
@spyrostz Do you have any opinion ?

Reply:
I know it is not a lot of data, but the less data we send, the better.
e.g. here: https://gridsingularity.atlassian.net/browse/D3ASIM-2227

Reply:
I would disagree on that, because these 2 have different meanings. No value means that the user has not configured it. ["House 1"]["export"]["peak_energy_kWh"] = 0 means that there was no export at all during the simulation, 100% self consumption

Reply:
discussed offline, no need for any further changes.